tracetypes/org.eclipse.tracecompass.incubator.opentracing.core/src/org/eclipse/tracecompass/incubator/opentracing/core/analysis/callstack/SpanCallGraphAnalysis.java

```diff
@@ -0,0 +1,178 @@
+package org.eclipse.tracecompass.incubator.opentracing.core.analysis.callstack;
+
+import java.util.Collection;
+import java.util.Collections;
+import java.util.List;
+
+import org.eclipse.core.runtime.IProgressMonitor;
+import org.eclipse.jdt.annotation.NonNull;
+import org.eclipse.tracecompass.incubator.analysis.core.concepts.ICallStackSymbol;
+import org.eclipse.tracecompass.incubator.analysis.core.concepts.ProcessStatusInterval;
+import org.eclipse.tracecompass.incubator.analysis.core.model.IHostModel;
+import org.eclipse.tracecompass.incubator.callstack.core.base.ICallStackElement;
```

Comment:
This should not depend on incubator callstack and should be dependent on the profiling core plugin, this comment can also be applied on the other added classes.